agimus_controller/agimus_controller/factory/robot_model.py

```diff
@@ -18,10 +18,12 @@ class RobotModelParameters:
     )  # Initial reduced configuration of the robot
     free_flyer: bool = False  # True if the robot has a free flyer
     locked_joint_names: list[str] = field(default_factory=list)
-    urdf_path: Path = Path()  # Path to the URDF file
-    srdf_path: Path | None = None  # Path to the SRDF file
+    urdf: Path | str | None = (
```

Comment:
```suggestion
    urdf: Path | str  = (
```

Reply:
It cannot be None, no?

Reply:
So you want it to default to an empty path or empty string?

Reply:
Empty string